### PR TITLE
shift kubevirt functions from drivers to kubevirt_helper.go

### PR DESCRIPTION
**Pull request by spai-px**
**What this PR does / why we need it**:
This PR moves the kubevirt code from drivers folder to test folder 

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: tests/kubevirt_helper.go [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+			if err != nil {
+				return false, err
+			}
+
+			err = RestartKubevirtVM(v.Name, v.Namespace, true)
+			if err != nil {
+				return false, err
+			}
+			log.InfoD("Sleep for 5mins for vm to come up")
+			time.Sleep(5 * time.Minute)
```
Comment (dbhatnagar-px) [inline tests/kubevirt_helper.go:74]: Note - in the next PR pls remove this hard sleep and work with ssh method instead.

File: tests/kubevirt_helper.go [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+			return "", err
+		}
+		return ScName.Name, nil
+	}
+	return "", fmt.Errorf("failed to get storage class of PVC attached to VM [%s] in namespace [%s]", vm.App.Key, vm.App.NameSpace)
+}
+
+// startAndWaitForVMIMigration starts the VM migration and waits for the VM to be in running state in the new node
+func StartAndWaitForVMIMigration(virtualMachine *scheduler.Context, ctx context1.Context) (bool, error) {
+
```
Comment (dbhatnagar-px) [inline tests/kubevirt_helper.go:131]: @spai-px : I see @pureneelesh reviewed this method already in the other PR. Please incorporate those comments here. 
Also I think the other PR is now no longer required right ?

Comment (spai-px) [reply tests/kubevirt_helper.go:131]: Ok will do.

**Comment by spai-px** [comment]
Closing this PR. All the changes have been made in [PR Link](https://github.com/portworx/torpedo/pull/2277)

**Comment by spai-px** [comment]
Closing this PR. All the changes have been made in https://github.com/portworx/torpedo/pull/2277

**Pull request closed without merging**